locallibrary/settings.py

```diff
@@ -133,21 +133,32 @@
 
 # Update database configuration from $DATABASE_URL environment variable (if defined)
 import dj_database_url
-db_from_env = dj_database_url.config(conn_max_age=500)
-DATABASES['default'].update(db_from_env)
 
+if 'DATABASE_URL' in os.environ:
+    DATABASES['default'] = dj_database_url.config(
+        conn_max_age=500,
+        conn_health_checks=True,
+    )
```

Comment:
FYI, this change needed because dj_database_url was updated to latest version, which gives a warning if the DATABASE_URL Is not defined. This is a valid case, and previously did not warn. I could have fixed it by created a default URL in the `config()` call but I prefer this approach (and have tested it).